src/Components/Common/Button/Button.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import './Button.scss';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import google from '../../../assets/images/google.png';
+import facebook from '../../../assets/images/facebook.png';
+import twitter from '../../../assets/images/twitter.png';
+
+/**
+ * Switch button icons
+ * @param {*} name
+ * @returns {object} Icon
+ */
+const setIcon = name => {
+  switch (name) {
+    case 'google':
+      return google;
+    case 'facebook':
+      return facebook;
+    case 'twitter':
+      return twitter;
+    default:
+      return null;
+  }
+};
+
+/**
+ * Change the element
+ * opacity
+ *
+ * @param {*} e
+ * @param {*} value
+ * @returns {void}
+ */
+const changeOpacity = (e, value) => {
+  e.target.style.opacity = value;
+};
+
+/**
+ * Shared button component
+ * @param {object} props
+ * @returns {object} Jsx component
+ */
+const Button = ({ social, text, color, full, size, disabled, icon, loading }) => {
```

Comment:
Good job! @gracelungu.

I like the way you handled the dynamic part of the component. However, it would be better if you add props that will be a function to execute some event on the button. For instance, the `onClick` props to handle  onClick event